frontend/src/main/scala/bloop/bsp/BloopBspServices.scala

```diff
@@ -795,7 +797,8 @@ final class BloopBspServices(
           Task.now(sys.error(s"Failed to run main class in $project due to: ${error.getMessage}"))
         case Right(mainClass) =>
           project.platform match {
-            case Platform.Jvm(config0, _, _, _, _) =>
+            case Platform.Jvm(compileConfig0, _, _, runtimeConfig0, _, _) =>
+              val config0 = runtimeConfig0.getOrElse(compileConfig0)
```

Comment:
I think it would be good to move this "runtimeConfig.getOrElse(compileConfig)" logic into a reusable method on `Platform.Jvm` to ensure it's used consistently in all places that need the "runtime config"

Reply:
Absolutely, I forgot that I wanted to do this and rename `jdkConfig`, which is easy to misuse at the moment 👍